manifest_template.yml

```diff
@@ -60,24 +66,21 @@ apigee:
         attributes:
           - name: access
             value: public
-            # For a parameterised example of rate limiting per environment,
-            # see https://github.com/NHSDigital/personal-demographics-service-api/blob/master/manifest_template.yml
           - name: ratelimiting
             value:
               {{ NAME }}:
```

Comment:
Should this be instead
{{ SERVICE_NAME }}-{{ VARIANT.service | default(VARIANT.name) }}-{{ ENV.name }}
According to this line in https://nhsd-confluence.digital.nhs.uk/pages/viewpage.action?spaceKey=APM&title=Rate+limiting+shared+flow#Ratelimitingsharedflow-Settingpolicyvariables

> my-proxy-internal-dev: # this controls the per-proxy rate limiting policies, AND MUST MATCH YOUR PROXY NAME